docs/en_us/platform_api/source/courses/blocks.rst

```diff
@@ -0,0 +1,347 @@
+.. _Courses API Blocks Resource:
+
+########################################
+Courses API Blocks Resource
+########################################
+
+With the Courses API **Blocks** resource, you can complete the
+following tasks.
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _Get a list of the course blocks in a course:
+
+****************************************
+Get a List of Course Blocks in a Course
+****************************************
+
+The endpoint to get a list of course blocks in a course is
+``/api/courses/v1/blocks/?course_id=<course_id>``.
```

Comment:
I can't remember if I already wrote this, sorry if duplicate. There's something about the query parameter that doesn't seem like a good fit with the word "endpoint." I'm surprised the course ID isn't a path segment (/api/courses/v1/blocks/{course_id}). I wanted to say this, but I'm also feeling that we don't need to worry about it now.